reticulator/reticulator.py

```diff
@@ -755,6 +769,52 @@ def __init__(self, key: str, value: str, comment: str = "") -> None:
         self.value = value
         self.comment = comment
 
+class FormatVersion:
+    def __init__(self, version) -> None:
+        if isinstance(version, FormatVersion):
+            self.major = version.major
+            self.minor = version.minor
+            self.patch = version.patch
+            return
+        elif isinstance(version, str):
+            elements = version.split('.')
+        else:
+            # Change to suitable error
+            raise TypeError()
```

Comment:
Should this be FormatVersionError?

Reply:
I thought TypeError, as they input the wrong type to be converted to a format version. Though FormatVersion('Invalid type!') may work?